bukkit/src/main/java/net/vyhub/command/Ban.java

```diff
@@ -78,19 +52,19 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
                         throw new NumberFormatException();
                     }
                 } catch (NumberFormatException nfe) {
-                    sender.sendMessage(getPlatform().getI18n().get("invalidNumberOfMinutes"));
+                    sender.sendMessage(platform.getI18n().get("invalidNumberOfMinutes"));
                     return false;
                 }
 
                 String reason = args.length == 3 ? args[2] : null;
-                p.kickPlayer(String.format(getPlatform().getI18n().get("youGotTimeBanned"), minutes, reason));
+                p.kickPlayer(String.format(platform.getI18n().get("youGotTimeBanned"), minutes, reason));
                 Bukkit.getBanList(BanList.Type.NAME).addBan(p.getUniqueId().toString(), reason, new Date(Calendar.getInstance().getTimeInMillis() + (minutes * 60 * 1000)), sender.getName());
             } else {
-                sender.sendMessage(getPlatform().getI18n().get("playerMustBeOnline"));
+                sender.sendMessage(platform.getI18n().get("playerMustBeOnline"));
```

Comment:
Is this the intended way to use i18n? Maybe calling `getI18n()` everytime causes performance issues?

Reply:
No, getters are as performant as direct access.